packages/cli/src/cmds/preview/preview.ts

```diff
@@ -741,7 +765,25 @@ export default class Preview extends YargsCommand {
       },
     ];
     cliLogger.necessaryLog(LogLevel.Info, utils.getColorizedString(message));
-    await open(sideloadingUrl);
+    switch (browser) {
+      case constants.Browser.chrome:
+        await open(sideloadingUrl, {
```

Comment:
What if the selected browser is not installed? Any `try-catch` or error handling logic?

Reply:
Good catch. Added error handling for this.